### ISSUE TITLE
Popup with CustomUI?

### ISSUE THREAD

**Issue by EmilAlipiev** [Samsung/Tizen.CircularUI#249]
Is it possible to display a popup with customUI, not only text and button? I couldnt find any example. Or Rg.Popup package is the only way we can get this job done?

**Comment by jkpu** [comment]
Current Information & TwoButton Popup are not support custom UI. which kind of property(e.g. background image , bg color , button position) do you want to add ?

**Comment by EmilAlipiev** [comment]
yes i would like to use image, label etc. I played around with rg.popup @myroot had PR for tizen which isnt merged into master but it works perfectly fine on mobile. However i am looking more for wearable.

**Comment by jkpu** [comment]
I have checked Rg.Plugins.Popup in https://github.com/rotorgames/Rg.Plugins.Popup/pull/272
This popup is only for mobile profile. 
In Tizen wearable device, Popup is displayed full screen only same as Information/TwoButtonPopup. partial displayed popup is not supported. 

How about you use page and NavigationPage instead of Popup? you can customize page and push (show popup) and then pop page when press button or back-button.

**Comment by rookiejava** [comment]
Refer to #276
